### DISCUSSION TITLE
Markdown tables

### DISCUSSION THREAD

**Discussion by johnnyggalt** [mealie-recipes/mealie#4702] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Until recently I have maintained my own git repo of recipes and such, and often include tables showing pricing information as at a certain date (so I can see how much I'm saving compared to buying a similar product from the supermarket or restaurant). Copy and pasting these tables into Mealie does not result in a nicely formatted table.

### Please provide a concise description of the feature that would resolve your issue.

Support for [Markdown tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables).

### Please consider and list out some caveats or tradeoffs made in your design decision

Not sure I understand the question here, since I haven't made a design decision 🙂 But the one thing I wondered is how prevalent table support actually is. It's worked pretty much everywhere I've ever needed tables in a Markdown context, but perhaps I've just been lucky...

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This actually is supported in most places, but the formatting is terrible. I can open a PR to fix this!

**Comment by michael-genson** [comment]
This is now fixed! https://github.com/mealie-recipes/mealie/pull/4703

**Comment by johnnyggalt** [comment]
@michael-genson Amazing, thank you!
